src/components/availability-label/availability-label.scss

```diff
@@ -0,0 +1,15 @@
+/* stylelint-disable */
+.check-icon {
```

Comment:
This will be a reoccuring question :)
Is this styling necessary? It is not defined i dpl-design-system?

Reply:
good point! I'll implement it in the design system instead :)